test/manifests/samples/extension-0-EPHEMERAL_ACCESS_vars.js

```diff
@@ -4,7 +4,8 @@
         // "EPHEMERAL_ACCESS_LABEL_VALUE": "some-label-value",
         "EPHEMERAL_ACCESS_MAIN_BANNER": "All production changes require an associated change request. Click the REQUEST ACCESS button above to automatically create a change request associated with your user",
         "EPHEMERAL_ACCESS_MAIN_BANNER_ADDITIONAL_INFO_LINK": "https://additional-info-link.com",
-        "EPHEMERAL_ACCESS_DEFAULT_ROLE": "devops"
+        "EPHEMERAL_ACCESS_DEFAULT_BASE_ROLE": "Read",
+        "EPHEMERAL_ACCESS_DEFAULT_TARGET_ROLE": "devops"
```

Comment:
Can you please update the sample as well?

Reply:
woops, updated that now!